index.js

```diff
@@ -78,6 +109,34 @@ class TrezorKeyring extends EventEmitter {
     return this.model;
   }
 
+  /**
+   * Gets the vendor, if known.
+   *
+   * @returns {"trezor" | "onekey"}
+   */
+  async getVendor() {
+    return this.vendor || (this.vendor = await this.fetchVendor());
+  }
+
+  /**
+   * fetch vendor by call getFeatures
+   * @param {object} features
+   * @returns {'onekey' | 'trezor' | null}
+   */
+  fetchVendor() {
+    return TrezorConnect.getFeatures()
+      .then((response) => {
+        if (!response.success) {
+          return null;
+        }
+        const vendor = getVendorName(response.payload);
+        return vendor;
```

Comment:
Nit: if you wanted, you could simplify this as:

``` suggestion
        return response.success ? getVendorName(response.payload) : null;
```

Reply:
I think the original way of writing is clearer and easier to use break point to found bugs, it is better to keep it as it is here.